modules/update-service-install-cli.adoc

```diff
@@ -0,0 +1,122 @@
+[id="update-service-install-cli"]
+= Installing the {product-title} Update Service using the CLI
+include::modules/common-attributes.adoc[]
+
+You can use the {product-title} CLI to install the {product-title} Update Service operator.
+
+== Procedure
+
+To install the {product-title} Update Service operator using the {product-title} CLI:
+
+. Create a Namespace for the {product-title} Update Service operator:
+
+.. Create a Namespace object YAML file (for example, `update-service-namespace.yaml`) for the {product-title} Update Service operator:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: FIXME-default-namespace
+  annotations:
+    openshift.io/node-selector: ""
+  labels:
+    openshift.io/cluster-monitoring: "true" <1>
+----
+<1> Set the `openshift.io/cluster-monitoring` label to enable operator recommended cluster monitoring on this namespace.
+
+.. Create the Namespace:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
++
+For example:
++
+[source,terminal]
+----
+$ oc create -f update-service-namespace.yaml
+----
+
+. Install the {product-title} Update Service operator by creating the following objects:
+
+.. Create an Operator Group object YAML file (for example, `update-service-operator-group.yaml`):
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: update-service-operator-group
+spec:
+  targetNamespaces:
+  - FIXME-default-operand-namespace
+----
+
+.. Create an Operator Group object:
++
+[source,terminal]
+----
+$ oc -n FIXME-default-namespace create -f <file-name>.yaml
+----
++
+For example:
++
+[source,terminal]
+----
+$ oc -n FIXME-default-namespace create -f update-service-operator-group.yaml
+----
+
+.. Create a Subscription object YAML file (for example, `update-service-subscription.yaml`):
++
+.Example Subscription
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: update-service-subscription
+spec:
+  channel: "{product-version}" <1>
+  installPlanApproval: "Automatic"
+  source: "redhat-operators" <2>
+  sourceNamespace: "openshift-marketplace"
+  name: "FIXME-update-service-operator"
+----
+<1> Specify `{product-version}` as the channel.
+<2> Specify `redhat-operators`. If your {product-title} cluster is installed on a restricted network, also known as a disconnected cluster, specify the name of the CatalogSource object created when you configured the Operator Lifecycle Manager (OLM).
```

Comment:
I think you need to explain what this is and the two options, something like:

```suggestion
<2> Specify the name of the Operator channel. For clusters that do not use a custom Operator Lifecycle Manager (OLM), specify `redhat-operators`. If your {product-title} cluster is installed on a restricted network, also known as a disconnected cluster, specify the name of the CatalogSource object created when you configured the OLM.
```

If it's not really simple to find this CatalogSource object name, I strongly suggest adding a note to the OLM configuration steps that says that you're going to need that name when you upgrade.

Reply:
Landed your hunk with adf5ca0732 -> 2c77ff9ce2.  I'm not sure about the "If it's not really simple..." bit.